index.cjs

```diff
@@ -759,7 +766,20 @@ safeql: try {
     !process.env.PGPASSWORD ||
     !process.env.PGDATABASE
   ) {
-    throw new Error('Environment variables are not set');
+    const missingEnvVars = [
+      'PGHOST',
```

Comment:
this list of env vars should not appear twice in the code

Reply:
Updated in this commit
[0152620](https://github.com/upleveled/eslint-config-upleveled/pull/234/commits/0152620e518db537b9b70882ab8e6737a6adc333)